workflow-templates/im-run-flyway-repair.yml

```diff
@@ -0,0 +1,100 @@
+# Workflow Code: SpiritedGnat_v1    DO NOT REMOVE
+
+# TODO: Ensure each of the repo-level and env-level secrets used in this workflow have been populated by an admin in your repository.
+
+name: Runbook - Flyway Repair
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: The environment the service is in - dev|qa|stage|prod|demo|uat # TODO:  Update for the environments that are available
+        required: true
+        default: dev
+
+env:
+  DB_SERVER_PORT: ''
+  MIGRATIONS_HISTORY_TABLE: ''
+  PATH_TO_MIGRATION_FILES: ''
+
+jobs:
+  set-vars:
+    runs-on: ubuntu-latest # Force this to run on github-hosted runner by using a tag that does not exist on self-hosted runners
+
+    # To use any of these outputs in another job use the following syntax: ${{ needs.set-vars.outputs.ENVIRONMENT }}
+    outputs:
+      ENVIRONMENT: ${{ steps.clean-env.outputs.ENVIRONMENT }}
+
+    steps:
+      # To use this value: ${{ needs.set-vars.outputs.ENVIRONMENT }}
+      - name: Set ENVIRONMENT
+        id: clean-env
+        uses: im-open/set-environment-variables-by-scope@v1.0.2
+        with:
+          scope: ${{ github.event.inputs.environment }}
+          create-output-variables: true
+          error-on-no-match: true
+          custom-error-message: 'The environment must be Dev, QA, Stage Demo, UAT or Prod' # TODO:  Update for the environments that are available
+        env:
+          # TODO:  Update for the environments your project contains
+          # The value array contains the environments it will match and the corresponding key is
+          # the environment it will output if one of the values was found.  It is case insensitive.
+          ENVIRONMENT@dev d development: 'dev'
+          ENVIRONMENT@qa q: 'qa'
+          ENVIRONMENT@stg s stage: 'stage'
+          ENVIRONMENT@o demo: 'demo'
+          ENVIRONMENT@u uat: 'uat'
+          ENVIRONMENT@prod p production: 'prod'
+
+      - run: echo "The current environment is ${{ steps.clean-env.outputs.ENVIRONMENT }}.  The event is ${{ github.event.inputs.eventName }}"
```

Comment:
Why go with the event name here? If you had multiple triggers I could see logging it out, but this one will probably only have the one trigger, right?